js/buildAll.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+from os.path import abspath, dirname
+
+build_script = "build.sh"
+if "win" in sys.platform:
+    build_script = "build.bat"
+
+build_dir = ["wasm", "wasm_SIMD", "wasm_threaded", "wasm_SIMD_threaded"]
+build_flags = ["", " --enable_wasm_simd", " --enable_wasm_threads", " --enable_wasm_simd --enable_wasm_threads "]
+wasm_file_names = ["ort-wasm.wasm", "ort-wasm-simd.wasm", "ort-wasm-threaded.wasm", "ort-wasm-simd-threaded.wasm"]
+js_file_names = ["ort-wasm.js", "ort-wasm-threaded.worker.js", "ort-wasm-threaded.js"]
+build_dir_js = ["wasm", "wasm_threaded", "wasm_threaded"]
+
+default_flags = " --build_wasm --skip_tests --skip_submodule_sync --parallel "
+BinariesDirectory = ".\\build\\"
```

Comment:
use consistent naming convention for consts. you have multiple in this file:

default_flags
BinariesDirectory
Dist_Path